Rakefile

```diff
@@ -13,7 +13,12 @@ require 'rspec/core/rake_task'
 # Rake::Application.send :include, TempFixForRakeLastComment
 # ### end of tempfix
 
-RSpec::Core::RakeTask.new(:spec)
+RSpec::Core::RakeTask.new(:spec) do |t, _task_args|
+  # Allow running a single spec `bundle exec rake spec SPEC=spec/features/threading_spec.rb`
+  if ENV["SPEC"]
+    t.rspec_opts = "#{ENV["SPEC"]}"
+  end
+end
```

Comment:
why not just run `bundle exec rspec spec/features/threading_spec.rb` ?

Reply:
🤦🏻‍♂️ yep you're right! I'll remove that

Reply:
All set, i removed the code